<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapOptionGeneral</class>
 <widget class="QWidget" name="MapOptionGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Layer Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLayerName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Layer Source</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLayerSource">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Layer Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLayerType">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>144</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinate Reference System</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCRS">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>120</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>EPSG:4326 - WGS 84</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxScaleDependentVisibility">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>143</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale dependent visibility</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelMaximumScale">
       <property name="text">
        <string>Maximum (inclusive)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxMaximumScale"/>
     </widget>
     <widget class="QSplitter" name="splitter_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelMinimumScale">
       <property name="text">
        <string>Minimum (exclusive)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxMinimumScale"/>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Source Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Observation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Field</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCRS_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ESRI Shapefile</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditObservation">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditField">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
